sdk/daml-lf/transaction-test-lib/src/main/scala/lf/transaction/test/TestNodeBuilder.scala

```diff
@@ -72,7 +74,7 @@ trait TestNodeBuilder {
 
     Node.Create(
       coid = id,
-      packageName = packageName.filter(_ => transactionVersion < TransactionVersion.minUpgrade),
+      packageName = packageName.filter(_ => transactionVersion >= TransactionVersion.minUpgrade),
```

Comment:
Why did this comparator change to the exact opposite? Was there a bug in the original line?

Reply:
It looks like a bug in the original code -- maybe a confusion between `filter` and `filterNot`. 
packageName should be define when `transactionVersion >= TransactionVersion.minUpgrade`.
cc @simonmaxen-da